x-pack/plugins/security_solution/server/endpoint/services/fleet/endpoint_fleet_services_factory.ts

```diff
@@ -82,21 +82,20 @@ export interface EndpointFleetServicesInterface {
   packagePolicy: PackagePolicyClient;
 }
 
-export interface EndpointScopedFleetServicesInterface extends EndpointFleetServicesInterface {
+export interface EndpointInternalFleetServicesInterface extends EndpointFleetServicesInterface {
   /**
    * get internal fleet services instance
    */
   asInternal: EndpointFleetServicesFactoryInterface['asInternalUser'];
+  /**
+   * An internal SO client (readonly) that can be used with the Fleet services that require it
+   */
+  internalReadonlySoClient: SavedObjectsClientContract;
 }
 
-export interface EndpointInternalFleetServicesInterface extends EndpointFleetServicesInterface {
+export interface EndpointScopedFleetServicesInterface extends EndpointFleetServicesInterface {
```

Comment:
I'm not sure I understand why these were all changed. Can you provide some clarity here?

Reply:
The type names are quite confusing relative to what the interface has (internal has the scoped client and vice versa) which is why I renamed them. I confirmed with @joshdover before renaming these.

Reply:
reverted 4f0ac7a73c260fe5b9c33ee65249ec9ba754d7aa